add_npm_package.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Run as ./add_npm_package.sh name version strategy
+program_exists() {
+  which "$@" &> /dev/null;
+}
+
+if !(program_exists crudini); then
+  echo "crudini is not installed - installing..."
+  sudo yum install crudini
+  echo "crudini installed..."
+fi
+
+if !(program_exists npm2rpm); then
+  echo "npm2rpm is not installed - installing..."
+  sudo npm install -g npm2rpm
+  echo "npm2rpm installed..."
+fi
+
+echo -n "Making directory..."
+mkdir nodejs-$1
+echo "FINISHED"
+echo -n "Creating specs and downloading sources..."
+npm2rpm -n $1 -v $2 -s $3
```

Comment:
Add a check at the top of the script for usage, `if [ $# -ne 3 ]; then` echo usage out.